src/gui/mzroll/peakdetectiondialog.cpp

```diff
@@ -227,29 +208,25 @@ void PeakDetectionDialog::closeEvent(QCloseEvent* event)
     emit updateSettings(peakSettings);
 }
 
-void PeakDetectionDialog::showSettingsForm() {
-
-    mainwindow->getAnalytics()->hitScreenView("OptionsDialog");    
-    mainwindow->settingsForm->exec();
-    mainwindow->settingsForm->setIsotopeDetectionTab();
-}
-
 void PeakDetectionDialog::dbOptionsClicked()
 {   
     if (dbOptions->isChecked()) {
         mainwindow->alignmentDialog->peakDetectionAlgo->setCurrentIndex(0);
         featureOptions->setChecked(false);
-        reportIsotopesOptions->setEnabled(true);
     } else {
         mainwindow->alignmentDialog->peakDetectionAlgo->setCurrentIndex(1);
         featureOptions->setChecked(true);
-        reportIsotopesOptions->setEnabled(false);
     }
     
     QString dbName = compoundDatabase->currentText();
     toggleFragmentation(dbName);
 }
 
+bool PeakDetectionDialog::getDbOptions()
```

Comment:
You don't have to make this change but the name of this method can be a bit better (suggestion: `isDatabaseSearchSelected`).

Reply:
or maybe `isDatabaseSearchEnabled`?